README.md

```diff
@@ -110,6 +153,27 @@ Usage
 3. Input the [Device Farm Run Configuration](https://docs.aws.amazon.com/devicefarm/latest/developerguide/test-runs.html#test-runs-configuration)
 4. Click "Generate Pipeline Script"
 
+## TroubleShooting
+
+# "Invalid Credentials" Error while validating Credentials
+
+We validate two things: 1) If the credentials are valid, 2) and if they have access to AWS Device Farm
+
+1. Log in to your jenkins host.
+2. If using access and secret key,
+Run "aws devicefarm list-projects" on the host running jenkins.
+If this fails, first check you credentials.Is they are correct, refer to the "Generating a proper IAM user" to ensure that you have given Device Farm necessary Credentials.
```

Comment:
If this fails, first check you credentials. <add space> If they are correct, refer to the "Generating a proper IAM user" to ensure that you have given Device Farm necessary credentials.

Reply:
Changed